src/components/comments/CommentEntries.jsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import PropTypes from 'prop-types';
 import './style.scss';
-import LikeNumberGroup from '../../components/LikeNumberGroup';
-import HorizontalLine from '../../components/HorizontalLine';
+import { LikeNumberGroup } from '../../components/like-button';
```

Comment:
@jsamchineme Javascript adopts the camelcase pattern for filenames, directory names, variables, etc. Please change `lines 4 and 5` to use this pattern.